apps/evm/src/clients/api/queries/useGetPools/__tests__/__snapshots__/index.prime.spec.ts.snap

```diff
@@ -2876,7 +2876,7 @@ exports[`useGetPools > filters out Prime simulations that are 0 1`] = `
                 "decimals": 6,
                 "symbol": "USDT",
               },
-              "type": undefined,
+              "type": "merkl",
```

Comment:
I'm in doubt about some of the mocks set as a `merkl` distribution type 🤔, might be worth double checking if the condition is right

Reply:
Yeah these ones are correct, I manually updated the mock data to make sure all reward distributors had a type, so that it reflects better what the API would respond (since technically it would never respond `undefined` for the type).